client-app/modules/news/components/news-article-content.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <VcWidget :title="newsArticle.title" class="news-article-content">
+    <VcMarkdownRender :src="newsArticle.content ?? ''" class="news-article-content__content" />
```

Comment:
Please add a blank line between tags to visually separate them. Here and in other files.